lightning/src/ln/channelmanager.rs

```diff
@@ -483,7 +483,7 @@ impl<'a> Drop for PersistenceNotifierGuard<'a> {
 /// we, or our watchtower, must check for them having broadcast a theft transaction).
```

Comment:
What is the timeout measured in, apples? This comment should have always had units lol